projects/1.21/assets/rar-compat/rarcompat/lang/zh_cn.json

```
@@ -0,0 +1,95 @@
+{
+  "tooltip.relics.umbrella.description": "减缓摔落，从任何高度摔下都不会受伤。若使用得当，还能当做一面可靠的盾牌，让敌人无法靠近。",
+  "tooltip.relics.umbrella.ability.glider": "软着陆",
+  "tooltip.relics.umbrella.ability.glider.description": "手持雨伞时逐渐减缓玩家的摔落速度，并随之降低摔落伤害。摔落速度越慢，抵消的伤害也越多，最高可完全抵消。",
+  "tooltip.relics.umbrella.ability.glider.stat.speed.title": "摔落速度减缓：”",
+  "tooltip.relics.umbrella.ability.glider.stat.speed.value": "%1$s%%每秒。",
+  "tooltip.relics.umbrella.ability.shield": "空气盾牌",
+  "tooltip.relics.umbrella.ability.shield.description": "使用时相当于盾牌，能够击退目标。",
+  "tooltip.relics.umbrella.ability.shield.stat.knockback.title": "击退力度：",
+  "tooltip.relics.umbrella.ability.shield.stat.knockback.value": "~%1$s格。",
+
+  "tooltip.relics.plastic_drinking_hat.description": "加速饮用速度，不只能迅速解渴，甚至还能恢复饥饿值。",
+  "tooltip.relics.plastic_drinking_hat.ability.drinking": "迅速饮用",
+  "tooltip.relics.plastic_drinking_hat.ability.drinking.description": "增加饮用速度。",
+  "tooltip.relics.plastic_drinking_hat.ability.drinking.stat.speed.title": "额外饮用速度：",
+  "tooltip.relics.plastic_drinking_hat.ability.drinking.stat.speed.value": "+%1$s%%。",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition": "饱腹啜饮",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.description": "饮用液体的同时恢复玩家的饥饿值。",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.stat.hunger.title": "恢复的饥饿值：",
+  "tooltip.relics.plastic_drinking_hat.ability.nutrition.stat.hunger.value": "%1$s点。",
+
+  "tooltip.relics.novelty_drinking_hat.description": "加速饮用速度，不只能迅速解渴，甚至还能恢复饥饿值。",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking": "迅速饮用",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.description": "增加饮用速度。",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.stat.speed.title": "额外饮用速度：",
+  "tooltip.relics.novelty_drinking_hat.ability.drinking.stat.speed.value": "+%1$s%%。",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition": "饱腹啜饮",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.description": "饮用液体的同时恢复玩家的饥饿值。",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.stat.hunger.title": "恢复的饥饿值：",
+  "tooltip.relics.novelty_drinking_hat.ability.nutrition.stat.hunger.value": "%1$s点。",
+
+  "tooltip.relics.snorkel.description": "能让你长时间在水下探索，而不必担心溺水。",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: Allows for longer underwater exploration without the risk of drowning.**

```suggestion
  "tooltip.relics.snorkel.description": "能让你在水下探索更久，而不必担心溺水。",
```

---

过会修的时候再看看（）